app/controllers/application.js

```diff
@@ -20,6 +23,10 @@ export default Controller.extend({
   shouldShowFooter: computed.and('isNotOnboarding', 'isNotViewingProjectTaskBoard'),
   shouldShowSpacer: computed.alias('isNotViewingProjectTaskBoard'),
 
+  _updateRoute: observer('currentRouteName', function() {
+    get(this, 'currentRoute').didTransition();
+  }),
```

Comment:
I can't say it's clear to me what this part does. I mean, I get the what, not clear on the why